Sources/PackageRegistryTool/PackageRegistryTool+Auth.swift

```diff
@@ -90,6 +90,28 @@ private func readpassword(_ prompt: String) throws -> String {
 
 extension SwiftPackageRegistryTool {
     struct Login: SwiftCommand {
+
+        static func authenticationStorageKey(for registryURL: URL) throws -> String {
```

Comment:
We need to update the "read" too: https://github.com/apple/swift-package-manager/blob/main/Sources/PackageRegistry/RegistryConfiguration.swift#L69-L72